test/fixtures/cookbooks/test/recipes/chefdk.rb

```diff
@@ -2,10 +2,12 @@
   action :install
   channel :stable
   version '0.5.1'
+  omnitruck_endpoint node['chef-ingredient']['omnitruck']['endpoint']
```

Comment:
I think we should use node property to set this and remove this resource attribute.